classes/mock/generator.php

```diff
@@ -559,32 +637,57 @@ protected function getNamespace($class)
         return '\\' . $this->getDefaultNamespace() . ($lastAntiSlash === false ? '' : '\\' . substr($class, 0, $lastAntiSlash));
     }
 
-    protected function getReturnType(\reflectionMethod $method)
+    protected function getReturnTypeCode(\reflectionMethod $method)
```

Comment:
This method should not be renamed as it is used by the [`visibility-extension`}(https://github.com/atoum/visibility-extension/blob/master/classes/mock/generator.php#L47)